common/microsoft-extensions-ai-integration/AIPromptIntegration/AIPromptIntegration/Program.cs

```diff
@@ -1,59 +1,57 @@
-using AIPromptIntegration.Components;
+﻿using AIPromptIntegration.Components;
 using Microsoft.Extensions.AI;
-using Azure;
-using OpenAI;
+
+// Optional model provider implementations:
+// Azure models
+using Azure; 
 using Azure.AI.OpenAI;
+// OpenAI models
+using OpenAI;
+// GitHub models
+using System.ClientModel; 
 
 var builder = WebApplication.CreateBuilder(args);
 
 // Add services to the container.
 builder.Services.AddRazorComponents()
-    .AddInteractiveServerComponents();
+	.AddInteractiveServerComponents();
 
 builder.Services.AddTelerikBlazor();
 
-#region Azure AI Inference Client registration
-
-//builder.Services.AddChatClient(
-//    new Azure.AI.Inference.ChatCompletionsClient(
-//        new Uri("https://models.inference.ai.azure.com"),
-//        new AzureKeyCredential("YOUR_AZURE_OPENAI_CREDENTIAL")
-//    ).AsChatClient("Phi-3.5-MoE-instruct"));
-
-#endregion Azure AI Inference Client registration
-
-#region OpenAI Client registration
-
-//builder.Services.AddSingleton(new OpenAIClient("YOUR_API_KEY"));
-
-//builder.Services.AddChatClient(services => services.GetRequiredService<OpenAIClient>().AsChatClient("YOUR_MODEL_NAME"));
-
-#endregion OpenAI Client registration
-
-#region Azure OpenAI Client registration
-
-//builder.Services.AddSingleton(new AzureOpenAIClient(
-//   new Uri("YOUR_AZURE_OPENAI_ENDPOINT"),
-//   new AzureKeyCredential("YOUR_AZURE_OPENAI_CREDENTIAL")));
-
-//builder.Services.AddChatClient(services => services.GetRequiredService<AzureOpenAIClient>().AsChatClient("gpt-4o-mini"));
-
-#endregion Azure OpenAI Client registration
-
-#region Ollama Chat Client registration
-
-//builder.Services.AddChatClient(new OllamaChatClient(new Uri("THE_URI_OF_YOUR_CLIENT"), "llama3.1"));
-
-#endregion Ollama Chat Client registration
+// You will need to set the endpoint and key to your own values
+// You can do this using Visual Studio's "Manage User Secrets" UI, or on the command line 💻:
+//   cd this-project-directory
+//   dotnet user-secrets set Endpoint https://YOUR-DEPLOYMENT-NAME.openai.azure.com
+//   dotnet user-secrets set ApiKey super-secret-api-key
+
+// 🌐 The Uri of your provider
+var endpoint = builder.Configuration["Endpoint"] ?? throw new InvalidOperationException("Missing configuration: Endpoint. See the README for details.");
+// 🔑 The API Key for your provider
+var apikey = builder.Configuration["ApiKey"] ?? throw new InvalidOperationException("Missing configuration: ApiKey. See the README for details.");
+// 🧠 The model name or azure deployment name
+var model = "YOUR_MODEL_NAME";
+
+// Replace the innerClient below with your preferred model provider 
+var innerClient = new OpenAIClient(
+			new ApiKeyCredential(apikey),
```

Comment:
Use tabs for spaces in the whole file.

Reply:
Addressed in a separate pr - https://github.com/telerik/blazor-ui/pull/325